profile/README.md

```diff
@@ -14,32 +14,29 @@
 Miden is the edge blockchain that grants applications the power to scale with public and private transactions. It aims at builders who want to create an ecosystem for private asset management and OTC-trading.
 
 ## Project structure
-This repository stores all the code for Polygon Miden. Everything is open source. 
+This repository stores all the code for Miden. Everything is open source. 
 
 | Repository              | Description |
 | ----------------------- | ----------- |
-| [Miden Base](https://github.com/0xMiden/miden-base)    | Core components of the Polygon Miden rollup. |
+| [Miden Base](https://github.com/0xMiden/miden-base)    | Core components of the Miden chain. |
 | [Miden Compiler](https://github.com/0xMiden/compiler)  | Compiler from MidenIR to Miden Assembly. |
 | [Miden Client](https://github.com/0xMiden/miden-client)| A reference Miden Client to be used by users. |
 | [Miden Node](https://github.com/0xMiden/miden-node)    | Miden Node to be used by the Miden operators. |
-| [Miden VM](https://github.com/0xMiden/miden-vm)        | STARK-based virtual machine for Polygon Miden rollup. |
-| [Miden Crypto](https://github.com/0xMiden/crypto)      | Cryptographic primitives used in Polygon Miden rollup. |
+| [Miden VM](https://github.com/0xMiden/miden-vm)        | STARK-based virtual machine for the Miden chain. |
+| [Miden Crypto](https://github.com/0xMiden/crypto)      | Cryptographic primitives used in the Miden chain. |
 
 ## Status
-The [public testnet](https://0xmiden.github.io/miden-docs/imported/miden-client/src/get-started/prerequisites.html) is up and running. You can interact with the Miden node using the Miden client. Currently, there are only predefined smart contracts and node scripts. As soon as there is the Miden Rust compiler, builders can start writing their own contracts.
-
-You can also use the Miden VM v0.9 locally and on the Miden [playground](https://0xpolygonmiden.github.io/examples/). For more detailed information, check out the Miden VM repository or our [Miden VM docs](https://0xmiden.github.io/miden-docs/imported/miden-vm/src/intro/main.html).
+The [public testnet](https://testnet.midenscan.com/) is up and running. You can interact with the Miden node using the Miden client. The easiest way to try Miden is by installing the [Miden Rust client](https://0xpolygonmiden.github.io/miden-docs/imported/miden-client/src/get-started/prerequisites.html). If you feel like exploring, try [our tutorials](https://0xpolygonmiden.github.io/miden-docs/imported/miden-tutorials/src/index.html). As soon as there is the Miden Rust compiler, builders can start writing their own contracts in Rust.
     
-If you want to use parts of Polygon Miden - like the Miden VM or AirScript - please contact [us](dschmid@polygon.technology).
+If you want to use Miden - please contact [us](dominik@miden.team).
```

Comment:
Let's use `hello@miden.team` here.

I'd also mention something about building on Miden, and maybe link to the pioneers program.